Lightbox.js

```diff
@@ -117,6 +117,17 @@ export default class Lightbox extends Component {
     }
   }
 
+  onLongPress = (...args) => {
+    let result;
+    if (this.props.onLongPress) {
+      result = this.props.onLongPress(...args);
+    } else {
+      result = () => {}; // default behaviour is "do nothing"
```

Comment:
Can we just set it to null instead of create a new function in every render?

Reply:
Good point, I think I was concerned about red screen of death if null was passed in but I just tested it and it seems to fail quietly.